Makefile

```diff
@@ -201,13 +208,35 @@ electron-installer-dmg: release/out/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$
 TARGETS += \
 	electron-installer-dmg \
 	electron-installer-app-zip
+PUBLISH_AWS_S3 += \
```

Comment:
Yeah, that's something else I've already done in my yet-to-be-PRed `Makefile` changes ;-)
I want to get all my little changes in first, before starting with the simplification-via-variables, as the latter touches almost every line.